gnosis/safe-cmm-app-react
# Feature/#176/deployed params

# :warning: Merge after #196 :warning: 

# Description

Closes #176 

Adds missing Deployed params values
![screenshot_2020-11-30_15-25-24](https://user-images.githubusercontent.com/43217/100677852-c01c9200-3320-11eb-91c6-c709a994e698.png)

# To Test
1. Make sure there's an active strategy
1. Go to strategies > active
1. Click on any strategy to expand the row
1. Click on `Deployed Params`

- [ ] All values should be filled
- [ ] Token amounts should match the initial strategy deposit
- [ ] Total value should show an approximate USD amount, based on GP prices

# Background

N/A
